core/include/drivers/rtc.h

```diff
@@ -0,0 +1,136 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2022 Microchip.
+ */
+
+#ifndef DRIVERS_RTC_H
+#define DRIVERS_RTC_H
+
+#include <tee_api_types.h>
+
+#define RTC_CORRECTION_FEATURE	BIT(0)
+
+#define TIME_INIT(year, month, day, hour, min, sec) \
+					{sec, min, hour, day, month, year, 0}
+
+struct optee_rtc_time {
```

Comment:
This constitutes the ABI provided by the PTA in a later patch. How can we know that this struct is enough? When comparing with `struct rtc_time` in Linux there's a few more fields.

Reply:
Yes, in the linux `rtc_time` struct, there is `yday` which is the year day and `is_dst` for daylight saving time. The `yday` member can be computed based on the other informations and `is_dst` is actually not used. I asked the maintainer about these fields and he says they can be left out.

Reply:
Ok thanks @jenswi-linaro !